client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -2020,6 +2020,15 @@ Undocumented.prototype.getHelpLinks = function( searchQuery, fn ) {
 	}, fn );
 };
 
+Undocumented.prototype.getQandA = function( searchQuery, fn ) {
+	debug( 'help-contact-qanda/ searchQuery {searchQuery}' );
+
+	return this.wpcom.req.get( '/help/qanda', {
+		query: searchQuery,
+		site: config( 'happychat_support_blog' ),
```

Comment:
I'm not entirely sure on the debug line, I was just following the same style as used in getHelpLinks